tests/snapshots/assets__check_file@tablex.typ-40.snap

```diff
@@ -4273,9 +4221,7 @@ input_file: tests/assets/tablex.typ
     // the same size as the last specified row.
     let last-row-size = if rows.len() == 0 {
       auto
-    } else {
-      rows.last()
-    }
+    } else { rows.last() }
```

Comment:
Maybe we should either put the two branch in a single line, or we completely expand them. Current status doesn't look good.